### PR TITLE
Add ggwebplot, summarize.for.webplot functions

### PR DESCRIPTION
**Pull request by okenk**
I think everything is working but let me know if there are any issues or key features missing. In reference to #43.

### CODE REVIEW

**Comment by okenk** [comment]
Hi! Just checking the status if this pull request.

**Comment by slucey** [comment]
Kiva,  thanks for staying on this.  We're overhauling our issue tracking and this will be towards the top.  Should be able to merge soon.

File: R/ecopath_plot.R
Comment (SarahJWeisberg) [inline R/ecopath_plot.R:1]: Found a typo in line 51 that is messing up the webplots - should be y1 = prey.y

**Comment by andybeet** [comment]
Pull request [90](https://github.com/NOAA-EDAB/Rpath/pull/90) resolves this.

**Pull request closed without merging**